mbuild/tests/test_utils.py

```diff
@@ -186,3 +187,32 @@ def test_has_ipython(self):
 
     def test_removed_func_error(self):
         RemovedFuncError("old_function", "new_function", "0.0.0", "0.1.0")
+
+
+class TestUtilsConversion(BaseTest):
+    def test_RB_to_OPLS_c5_not_0(self):
+        with pytest.raises(
+            ValueError,
+            match=r"ERROR: c5 must equal zero, so this conversion is not possible.",
+        ):
+            c0 = 0.1
+            c1 = 0.1
+            c2 = -0.2
+            c3 = -0.1
+            c4 = -0.2
+            c5 = 0.3
+            RB_to_OPLS(c0, c1, c2, c3, c4, c5)
+
+    def test_RB_to_OPLS_f0_not_0(self):
+        with pytest.raises(
+            ValueError,
+            match=r"ERROR: f0 \= 2 \* \(c0 \+ c1 \+ c2 \+ c3 \+ c4 \+ c5\) must equal zero, "
+            "so this conversion is not possible.",
+        ):
+            c0 = 0.4
+            c1 = 0.4
+            c2 = -0.1
+            c3 = 0.4
+            c4 = -0.2
+            c5 = 0
+            RB_to_OPLS(c0, c1, c2, c3, c4, c5)
```

Comment:
We should probably also add a test showing that converting the RB values in gromacs [ffbonded.itp](https://github.com/gromacs/gromacs/blob/master/share/top/oplsaa.ff/ffbonded.itp) gives back the original opls parameters they're drawn from. Just as an additional proof of concept that the equation we have works in this situation.
We can use the class CT-CT-CT-CT from the paper with opls params V1=6.622, V2=0.948, V3=-1.388, and V4=-2.118 (doi:10.1021/jp004071w). 
If we pop in the values of this class from oplsaa.xml they should convert within our tolerances.
<Proper class1="CT" class2="CT" class3="CT" class4="CT" c0="2.9288" c1="-1.4644" c2="0.2092" c3="-1.6736" c4="0.0" c5="0.0"/> from line 2365 in oplsaa.xml.

Reply:
We could do this.  However, scanning and comparing the difference between the RB-torsion to OPLS is more proof IMO.  Noting they will fail if they are not convertible (i.e., c5=0 or sum of all constants not equal 0), which we would have to account for.